telemetry/definitions/commonDefinitions.json

```diff
@@ -5777,6 +5787,21 @@
                 }
             ]
         },
+        {
+            "name": "experiment_feature_config",
+            "description": "AB Testing Feature response and Cohort Assignments",
+            "metadata": [
+                {
+                    "type": "featureValue"
```

Comment:
can we reuse the existing featureId field: https://github.com/ivikash/aws-toolkit-common/blob/09e9c4114fcd7659fa3dc1f8863de94c01058ed3/telemetry/definitions/commonDefinitions.json#L1199-L1206

Reply:
Discussed with Justin and Keeping it as `id`.